sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/ClientCertificateInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.WebPubSub.AspNetCore
+{
+    /// <summary>
+    /// Client certificate info.
+    /// </summary>
+    public sealed class ClientCertificateInfo
```

Comment:
Do we expose this type publicly?

Reply:
Yes, because in the `ConnectEventRequest`, customer may need to pick up a cert from clients to authenticate the client.